cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -727,75 +639,75 @@ func (c *client) WaitForResourceStatuses() error {
 	})
 }
 
-func (c *client) waitForClusterDelete(namespace string) error {
-	return util.PollImmediate(retryIntervalResourceDelete, timeoutResourceDelete, func() (bool, error) {
-		klog.V(2).Infof("Waiting for Clusters to be deleted...")
-		response, err := c.clientSet.ClusterV1alpha2().Clusters(namespace).List(metav1.ListOptions{})
-		if err != nil {
-			return false, nil
-		}
-		if len(response.Items) > 0 {
-			return false, nil
-		}
-		return true, nil
-	})
-}
-
-func (c *client) waitForMachineDeploymentsDelete(namespace string) error {
-	return util.PollImmediate(retryIntervalResourceDelete, timeoutResourceDelete, func() (bool, error) {
-		klog.V(2).Infof("Waiting for MachineDeployments to be deleted...")
-		response, err := c.clientSet.ClusterV1alpha2().MachineDeployments(namespace).List(metav1.ListOptions{})
-		if err != nil {
-			return false, nil
-		}
-		if len(response.Items) > 0 {
-			return false, nil
-		}
-		return true, nil
-	})
-}
-
-func (c *client) waitForMachineSetsDelete(namespace string) error {
-	return util.PollImmediate(retryIntervalResourceDelete, timeoutResourceDelete, func() (bool, error) {
-		klog.V(2).Infof("Waiting for MachineSets to be deleted...")
-		response, err := c.clientSet.ClusterV1alpha2().MachineSets(namespace).List(metav1.ListOptions{})
-		if err != nil {
-			return false, nil
-		}
-		if len(response.Items) > 0 {
-			return false, nil
-		}
-		return true, nil
-	})
-}
-
-func (c *client) waitForMachinesDelete(namespace string) error {
-	return util.PollImmediate(retryIntervalResourceDelete, timeoutResourceDelete, func() (bool, error) {
-		klog.V(2).Infof("Waiting for Machines to be deleted...")
-		response, err := c.clientSet.ClusterV1alpha2().Machines(namespace).List(metav1.ListOptions{})
-		if err != nil {
-			return false, nil
-		}
-		if len(response.Items) > 0 {
-			return false, nil
-		}
-		return true, nil
-	})
-}
-
-func (c *client) waitForMachineDelete(namespace, name string) error {
-	return util.PollImmediate(retryIntervalResourceDelete, timeoutResourceDelete, func() (bool, error) {
-		klog.V(2).Infof("Waiting for Machine %s/%s to be deleted...", namespace, name)
-		response, err := c.clientSet.ClusterV1alpha2().Machines(namespace).Get(name, metav1.GetOptions{})
-		if err != nil {
-			return false, nil
-		}
-		if response != nil {
-			return false, nil
-		}
-		return true, nil
-	})
-}
+// func (c *client) waitForClusterDelete(namespace string) error {
```

Comment:
Might as well delete these, can always refer back to the `release-0.1` branch if we need to bring them back.